app/controllers/passengers_controller.rb

```diff
@@ -48,8 +48,11 @@ def destroy
   private
 
   def passenger_params
-    params.require(:passenger).permit(:name, :address, :email, :phone,
-      :wheelchair, :active, :permanent, :expiration, :note)
+    permitted_params = params.require(:passenger).permit(:name, :address, :email, :phone,
+        :wheelchair, :active, :permanent, :expiration, :note)
+    unless @current_user.admin?
+      permitted_params = permitted_params.except(:active, :permanent)
+    end
```

Comment:
I thought it was returning the params that were permitted for people who are dispatchers ... what is it actually returning?